src/tripCard/TripCard.style.tsx

```diff
@@ -26,6 +26,7 @@ export const StyledTripCard = styled(Card)`
 
   & .kirk-item.kirk-tripCard-top-item {
     padding-top: 0;
+    padding-left: 0 !important;
```

Comment:
Ouch! I'm sad to see this, if we had time to finish the normalization we wouldn't have to do this :/
Normally, a `Card` should have a 8px padding (not 16px) and 8px on the Item and other components inside it (the one you removed).

Reply:
`TripCard` might be the most complex component to normalize as it aggregates lots of others :/